src/frontend/components/Sidebar/Sidebar.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+
+import "./Sidebar.css"
+
+const Sidebar = () => {
+
+    const optionSelected = ({ isActive }) => {
+        return "sidebar__option" + (isActive ? "--selected" : "") + " flex--row";
+    }
+
+    return (
+        <>
+            <nav className="sidebar__container flex--column primary__font">
```

Comment:
Is there a reason to wrap your nav within a fragment? Since nav is already a parent, I think you can remove this.